### DISCUSSION TITLE
Can you add a gesture option that maximizes to 90% by default?

### DISCUSSION THREAD

**Discussion by YonglinChann** [chrenn/swish-dl#8] [Ideas]
Can you add a gesture option that maximizes to 90% by default?

Currently the default effect of the [swipe up once] option is "fill the entire screen", but in macOS Ventura, when Stage manager is turned on, filling the screen often causes trouble when switching windows, while keeping it maximized to 90% gives a clear view of all other windows on the left side of the screen.

**Comment by chrenn** [comment]
No, sorry. Stage Manager support is coming soon tho.

**Reply by YonglinChann** [reply]
I just read in the latest schedule about the new features planned for the Stage manager, too, and I'm really looking forward to the new version!
